src/components/decimal/decimal.test.tsx

```diff
@@ -1449,3 +1449,25 @@ test("the `required` prop is not passed to the hidden input", () => {
 
   expect(screen.getByTestId("hidden-input")).not.toBeRequired();
 });
+
+test("component should render without invariant firing in strict mode", () => {
+  configure({ reactStrictMode: true });
```

Comment:
comment: As this is setting a global configuration I think we should call `configure({ reactStrictMode: false })` at the end of the `test`.